templates/artist/artist_detail.html

```diff
@@ -24,23 +24,30 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    {% thumbnail artist.photo.img "500" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
-    {% endthumbnail %}
+    <div class="extra-padding">
+        {% thumbnail artist.photo.img "500" as thumb %}
+            <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
+        {% endthumbnail %}
+    </div>
+    <!-- Genre -->
     {% with genres=artist.genres.all %}
     <div class="break">
         <hr>
     </div>
-        <p>Genre{{ genres|length|pluralize }}: {{ genres|join:" / " }}</p>
+    <div class="uppercase">
```

Comment:
We should have `<div>` wrap around end template tags. It might be better to move `<div class="uppercase">` before the {% with %} template tag opens.